inotify_poller.go

```diff
@@ -119,10 +119,6 @@ func (poller *fdPoller) wait() (bool, error) {
 				}
 			}
 			if event.Fd == int32(poller.pipe[0]) {
-				if event.Events&unix.EPOLLHUP != 0 {
-					// Write pipe descriptor was closed, by us. This means we're closing down the
-					// watcher, and we should wake up.
-				}
```

Comment:
The comment says it "should wake up", but it doesn't do anything. I'm not sure if this can just be removed, or if there's an omission here and we should put something in this branch.